main.js

```diff
@@ -17,16 +16,15 @@ var entireMealOption = document.querySelector('#entire-meal-option').value;
 var cookpotImage = document.querySelector('.cookpot-image');
 var youShouldMake = document.querySelector('.you-should-make');
 var outputRecipe = document.querySelector('.output-recipe');
-var recipeText = document.querySelector('#recipe-text');
+var favsList = document.querySelector('.favs-list');
+var backButton = document.querySelector('.back-button');
+var favButton = document.querySelector('.fav-button');
+var clearButton = document.querySelector('.clear-button');
 
+viewFavsButton.addEventListener('click', showFavorites);
 letsCookButton.addEventListener('click', getCooking);
-
-function getCooking() {
-  event.preventDefault();
-  hide(cookpotImage);
```

Comment:
Broke up the getCooking function to make sure that it wasn't showing a new random recipe when it was called with the back button.